### PR TITLE
✨ dcc dashboard placeholder

### PR DESCRIPTION
**Pull request by hlminh2000**
**Description of changes**

Removes Donkey Kong :(

**Type of Change**

- [ ] Bug
- [ ] Styling
- [ ] New Feature

**Checklist before requesting review:**

- [ ] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [ ] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

File: pages/submission/dcc/dashboard.tsx [hunk 1]
```diff
@@ -1,68 +1,8 @@
...
-─────▀██▄───▀▀▀▀▀▀▀▀▀───▄██▀─────
-────────▀██▄▄───────▄▄██▀────────
-───────────▀▀███████▀▀───────────
-`}
-        </pre>
-      }
-    />
-  );
-});
+})(DccDashboard);
```
Comment (jaserud) [inline pages/submission/dcc/dashboard.tsx]: :hocho: :hocho: :hocho: :hocho: :hocho:

File: components/pages/submission-system/dcc/index.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import SubmissionLayout from '../layout';
+import { css } from '@emotion/core';
+import Typography from 'uikit/Typography';
+
+export default () => {
+  return (
+    <SubmissionLayout
+      subtitle="All Programs"
```
Comment (jaserud) [inline components/pages/submission-system/dcc/index.tsx]: this is probably a designer question but would this be `All Programs` ??

Comment (hlminh2000) [reply components/pages/submission-system/dcc/index.tsx]: fixed!

Comment (jaserud) [reply components/pages/submission-system/dcc/index.tsx]: oh I didn't really request a change.. but :+1:

**Pull request merged**